bert_mlm_example/losses.py

```
@@ -0,0 +1,15 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+class VAELoss(nn.Module):
+    """VAE loss combining reconstruction CE and KL divergence."""
+
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, logits, labels_onehot, mu, logvar, kl_weight=1.0):
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)